applications/nrf_desktop/src/modules/Kconfig.usb_state

```diff
@@ -178,6 +178,23 @@ config USBD_HID_IN_BUF_COUNT
 	  nRF Desktop queues HID reports at the source. There is no need to use
 	  multiple buffers in the IN pool per HID instance.
 
+config DESKTOP_USB_STACK_NEXT_DISABLE_ON_VBUS_REMOVAL
+	bool
+	default y if SOC_SERIES_NRF54HX
+	depends on DESKTOP_USB_INIT_THREAD
+	select REBOOT
+	help
+	  Disable USB stack on VBUS removal. This is a workaround for the USB
+	  driver not working correctly on the nRF54HX SoC. After the USB cable
+	  is removed, the USB driver is powered down and doesn't call appropriate
+	  callbacks to the USB stack. It may lead to the USB stack ending in the
+	  broken state. Calling usbd_disable() on VBUS removal workarounds this
+	  issue. When this option is enabled, the usb_init_thread will be used to
+	  call usbd_enable() and usbd_disable() functions on VBUS events. This is
+	  done because the usbd_enable() function blocks if USB cable is not
+	  connected. A separate thread allows application to trigger reboot from
+	  the workqueue context to prevent blocking application forever.
```

Comment:
```suggestion
	  Disable USB stack on VBUS removal. This is a workaround for the USB
	  driver not working correctly on the nRF54HX SoC. After the USB cable
	  is removed, the USB driver is powered down and doesn't call appropriate
	  callbacks to the USB stack. It may lead to the USB stack ending in a
	  broken state. Calling usbd_disable() on VBUS removal workarounds this
	  issue. When this option is enabled, the usb_init_thread will be used to
	  call the usbd_enable() and usbd_disable() functions on VBUS events. This is
	  done because the usbd_enable() function blocks if the USB cable is not
	  connected. A separate thread allows the application to trigger a reboot from
	  the workqueue context to prevent blocking the application forever.
```